modules/net-lb-app-ext/variables.tf

```diff
@@ -88,6 +88,21 @@ variable "labels" {
   default     = {}
 }
 
+variable "mtls_policy_config" {
+  description = "Configuration for mTLS for the HTTPS Load Balancer."
+  type = object({
+    client_validation_mode            = optional(string, "CLIENT_VALIDATION_MODE_UNSPECIFIED")
```

Comment:
if this is an enum with a finite set  of values, we usually check it via a validation rule on the variable